packages/samples/mvp-dapp/README.md

```diff
@@ -123,6 +123,11 @@ Follow the [Guide](../../../README.md) to build libraries developed in other wor
 $ npm run build
 ```
 
+### Start CKB Node
+
+- locally: https://docs.nervos.org/docs/basics/guides/testnet/
+- remotely: https://testnet.ckb.dev/rpc
```

Comment:
I think it is hard to understand, the first one is to tell developers how to start a node and find the rpc url, but the second one is just to tell developers the rpc url of testnet. If I were a fresh developer to use this, I would just type the `https://testnet.ckb.dev/rpc` to browser to find the rpc url first.

Reply:
What if name this section `Get CKB Node ready`, and add 2 sub-sections
1. How to run CKB node locally
2. Available CKB nodes of community